java-jacoco/src/main/java/org/sonar/plugins/jacoco/JaCoCoSensor.java

```diff
@@ -125,10 +125,12 @@ private void addAnalysisWarning(String format, Object... args) {
     return reportPaths;
   }
 
-  private static Optional<File> getReport(Configuration settings, FileSystem fs, String reportPathPropertyKey, String msg) {
+  private Optional<File> getReport(Configuration settings, FileSystem fs, String reportPathPropertyKey, String msg) {
     Optional<String> reportPathProp = settings.get(reportPathPropertyKey);
     if (reportPathProp.isPresent()) {
-      warnUsageOfDeprecatedProperty(settings, reportPathPropertyKey);
+      if (!settings.hasKey(REPORT_PATHS_PROPERTY)) {
```

Comment:
:warning: you should also fix this :bug: by replacing `.hasKey(REPORT_PATHS_PROPERTY)` by `.hasKey(reportPathPropertyKey)`